src/picongpu/include/particles/particleToGrid/derivedAttributes/LarmorPower.hpp

```diff
@@ -55,16 +55,19 @@ namespace derivedAttributes
         const float_X gamma2 = gamma * gamma;
         const float_X c2 = SPEED_OF_LIGHT * SPEED_OF_LIGHT;
 
-        const float3_X mom_dt = mom - mom_mt1;
+        const float3_X mom_dt = (mom - mom_mt1) / float_X(DELTA_T);
         const float_X el_factor = charge * charge
             / (float_X(6.0) * PI * EPS0 *
-               c2 * c2 * SPEED_OF_LIGHT * mass * mass);
-        const float_X energyLarmor = el_factor * gamma2 * gamma2
-            * (math::abs2(mom_dt) -
-               math::abs2(math::cross(mom, mom_dt)));
+               c2 * SPEED_OF_LIGHT * mass * mass) * gamma2 * gamma2;
```

Comment:
If you have a look at the [non-coveriant form of the Larmor power](https://en.wikipedia.org/wiki/Larmor_formula#Non-covariant_form) (Jackson 3rd Ed. Eq. 14.26) it goes with gamma^6 for beta^2. Here we use momentum^2 which is proportional to gamma^2, thus only gamma^4 remains in the prefactor.

@Heikman How do you derive an equation with gamma^2 ?